File: app/src/main/java/ar/com/wolox/android/example/ui/example/ExampleActivity.java
package ar.com.wolox.android.example.ui.example;


import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

import javax.inject.Inject;

public class ExampleActivity extends WolmoActivity {

    @Inject ExampleFragment mExampleFragment;

    @Override
    protected int layout() {
        return R.layout.activity_base;
    }

    @Override
    protected void init() {
        replaceFragment(R.id.activity_base_content, mExampleFragment);
    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/example/ExampleActivity.kt
package ar.com.wolox.android.example.ui.example

import ar.com.wolox.android.R
import ar.com.wolox.android.databinding.ActivityBaseBinding
import ar.com.wolox.wolmo.core.activity.WolmoActivity

class ExampleActivity : WolmoActivity<ActivityBaseBinding>() {

    override fun layout() = R.layout.activity_base

    override fun init() {
        replaceFragment(binding.activityBaseContent.id, ExampleFragment.newInstance())
    }
}
